docs/source/examples.rst

```diff
@@ -308,6 +308,35 @@ Plotting with Manim
             two_pi.next_to(label_coord, RIGHT + UP)
             self.add(func_graph, func_graph2, vert_line, graph_lab, graph_lab2, two_pi)
 
+
+.. manim:: ArgMinExample
+
+   class ArgMinExample(Scene):
+       def construct(self):
+           ax = Axes(
+               x_range=[0, 10], y_range=[0, 100, 10], axis_config={"include_tip": False}
+           )
+           labels = ax.get_axis_labels(x_label="x", y_label="f(x)")
+           
+           stri = r"{\underset {x\in S}{\operatorname {arg\,min} }}\,f(x):=\{x\in S~:~f(s)\geq f(x){\text{ for all }}s\in S\}"
+           self.add(MathTex(stri).scale(0.8).to_corner(UR))
```

Comment:
This string feels a little overwhelming to include (both visually and in terms of code) especially since it's intended for the example gallery. Perhaps it'd be better to remove or adjust it?

Reply:
Yeah, good point!
My initial idea was that this example shows on the one hand a mathematical function, and on the other hand the transfer of it to in manim.
But you are right, it can get removed, as this example gallery should also be for people who do not study math :)